pkg/manifests/manifests.go

```diff
@@ -38,7 +38,7 @@ func syncRepo(url, branch, deployKeyPath, relativeRoot string) (string, string,
 
 	if err != nil {
 		closer()
```

Comment:
Mind refactoring to propagate error all the way to `cobra.Command.RunE` instead of doing `Fatalf`? That would help with #33.

Reply:
Done!

This makes a 1 line change now take 45 minutes because of a +85/-55 collateral change. I'm happy to help, but I don't think a review comment on an unrelated change is the way to go about it.

Reply:
Looks good, thanks for help improving this :+1: